src/Bonus.java

```diff
@@ -13,23 +13,31 @@ public static void main(String[] args) {
         HashSet<Integer> secretnumber = randomnumbergenerator();
         String stringnumber = setToStringConverter(secretnumber);
         System.out.println(stringnumber);
-        feedback(/*vul hier het juiste argument in*/);
+        feedback(stringnumber);
 
     }
 
-    public static void/*moet dit returntype "void" zijn of wat anders?*/ randomnumbergenerator(/*Heeft deze methode nog parameter(s) nodig?*/){
+    public static HashSet<Integer> randomnumbergenerator(){
          /*
         Vul hier de body van de methode in.
-
         Stappenplan:
         - Maak een nieuwe variabele van type Random. (Tip: Zoek op internet hoe je Random kunt gebruiken)
         - Maak een nieuwe variabele van type HashSet.
         - Schrijf een while-loop om 4 random nummers aan de hashset toe te voegen
         - return de hashset
          */
+        Random rand = new Random();
+        int upperbound = 10;
+        HashSet<Integer> set = new HashSet<>();
```

Comment:
"set" is een nietszeggende naam. Probeer alle variabelen een betekenisvolle naam te geven.